shared/middleware/helpers/bucket_helper.rb

```diff
@@ -24,7 +24,7 @@ def category_from_file_type(extension)
     mime_type = Sinatra::Base.mime_type(extension)
     if mime_type == 'application/pdf'
       'pdf'
-    elsif ['.doc', '.docx'].include? extension
```

Comment:
Context is allowed file upload types. As of this change, we'll allow users to upload pdf, ms word docs, and plaintext